frontend/lib/app.tsx

```diff
@@ -198,8 +195,9 @@ export class AppWithoutRouter extends React.Component<AppPropsWithRouter, AppSta
         <Route path={Routes.login} exact component={LoginPage} />
         <Route path={Routes.adminLogin} exact component={LoginPage} />
         <Route path={Routes.logout} exact component={LogoutPage} />
-        <Route path={Routes.onboarding.prefix} component={LoadableOnboardingRoutes} />
+        {getOnboardingRouteForIntent(OnboardingInfoSignupIntent.LOC)}
         <Route path={Routes.loc.prefix} component={LoadableLetterOfComplaintRoutes} />
+        {getOnboardingRouteForIntent(OnboardingInfoSignupIntent.HP)}
         <Route path={Routes.hp.prefix} component={LoadableHPActionRoutes} />
```

Comment:
I am not a fan of how order-dependent these route definitions are, since the HP onboarding routes need to be defined before the other HP action routes or else they'll be shadowed by them, but I can't think of a better solution now since we want to lazily load the two sets of routes separately.